extensions/librdkafka/PublishKafka.cpp

```diff
@@ -141,7 +150,7 @@ bool PublishKafka::configureNewConnection(const std::shared_ptr<KafkaConnection>
 
   if (!key->client_id_.empty()) {
     rd_kafka_conf_set(conf_, "client.id", key->client_id_.c_str(), errstr, sizeof(errstr));
-    logger_->log_debug("PublishKafka: client.id [%s]", value);
+    logger_->log_debug("PublishKafka: client.id [%s]", key->client_id_.c_str());
```

Comment:
you can simply use key->client_id_ as we perform this conversion for you.

Reply:
thanks, I did not notice that